frontend/public/components/utils/documentation.tsx

```diff
@@ -1,3 +1,8 @@
+import { FLAGS } from '../../const';
 // Prefer the documentation base URL passed as a flag, but fall back to the latest docs if none was specified.
 export const openshiftHelpBase =
   window.SERVER_FLAGS.documentationBaseURL || 'https://docs.okd.io/latest/';
+
+export const networkPolicyDocs = FLAGS.OPENSHIFT
```

Comment:
This check won't work here. `FLAGS.OPENSHIFT` will always evaluate to true since it's just a constant string value.

https://github.com/spadgett/console/blob/7db7191536ad7951565b82665b7a36e86c5f19e8/frontend/public/const.ts#L50

You need to use `connectToFlags` in the component.